src/rosdep2/platforms/osx.py

```diff
@@ -204,7 +204,11 @@ def is_installed(r):
             pkg_info = pkg_info[0]
             linked_version = pkg_info['linked_keg']
             if not linked_version:
-                return False
+                keg_only = pkg_info['keg_only']
+                if keg_only:
+                    return True
```

Comment:
I think this code will `return True` for any keg-only formula, even if it's not actually installed. Do you want to use the `installed` field instead?

note that the `caveats` field in the json does not match the full output printed by `brew info`. I believe running `brew ruby -e 'require "caveats"; puts Caveats.new("qt@5".f).keg_only_text'` can generate the full caveats string